src/test/ui/traits/trait-with-dst-2.rs

```diff
@@ -0,0 +1,20 @@
+struct VTable<DST: ?Sized> {
+    _to_dst_ptr: fn(*mut ()) -> *mut DST,
+}
+
+trait HasVTableFor<'a, DST: ?Sized 'a> {
```

Comment:
Can you check whether fixing the missing comma here causes an ICE or not on 2018-10-13?
If not then let's remove the test and close the issue.

Reply:
Hmm, sorry but what's the meaning of missing comma? Do you mean `?Sized + 'a`?

Reply:
No I meant `<'a, DST: ?Sized, 'a>` but `+` is probably what was intended. :)

Reply:
Okay, I tested with `+` and comma but can't reproduce an ICE. (`nightly-2018-10-22` ~ `nightly-2018-10-24`)

Reply:
Alright; let's remove and close then.